packages/zowe-explorer/__tests__/__unit__/trees/uss/USSTree.unit.test.ts

```diff
@@ -861,6 +861,7 @@ describe("USSTree Unit Tests - Function filterBy", () => {
         const blockMocks = await createBlockMocks();
         globalMocks.isUsingTokenAuth.mockResolvedValueOnce(true);
         const sessionWithCred = createISession();
+        globalMocks.mockProfilesInstance.loadNamedProfile.mockReturnValue({ name: "ussTestSess2" } as any);
```

Comment:
I think we should try to fully stub test profiles (when possible) to ensure proper processing for the tree node logic:

```suggestion
        globalMocks.mockProfilesInstance.loadNamedProfile.mockReturnValue(createIProfile());
```

This ensures that name, type, etc. are present in the profile object before a function tries to do logic on it. Ran into a similar issue with tests missing that data in #3727

Reply:
addressed but a bit diff since test looking for particular profile name